app/components/NavButtons.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+import { useState } from 'react';
+import Link from 'next/link'
+import styled from 'styled-components';
+import tw from 'tailwind-styled-components';
+
+
+export default function NavButtons() {
+  const [navItems] = useState(['coins', 'portfolio']);
+  const [selectedItem, setSelectedItem] = useState("coins");
+
+  const handleClick = (item: any) => {
+    setSelectedItem(item);
+  };
+const width = styled.div`
+width:320px;
+`
+  const Wrapper = tw(width)`
+  flex
+  bg-black
+  bg-opacity-50
+  rounded-3xl  
+`;
```

Comment:
don;t put the styled components in the component

Reply:
oh do you mean the function description? should I declare this outside of the Navbuttons component ?